### PR TITLE
Support showing 16kb page size summary in chart page.

### PR DESCRIPTION
**Pull request by zhaobozhen**

### CODE REVIEW

File: app/src/main/kotlin/com/absinthe/libchecker/features/chart/impl/PageSize16KBChartDataSource.kt [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        if (!isActive) {
+          return@withContext
+        }
+        try {
+          val pi = PackageUtils.getPackageInfo(item.packageName)
+          if (pi.is16KBAligned()) {
+            classifiedList[SUPPORT_16KB].add(item)
+          } else {
+            classifiedList[NOT_SUPPORT_16KB].add(item)
+          }
```
Comment (JunkFood02) [inline app/src/main/kotlin/com/absinthe/libchecker/features/chart/impl/PageSize16KBChartDataSource.kt:48]: just to clarify, apps that are entirely written in Java or Kotlin (with no native code) are inherently compatible with 16KB page size, as the ART will handle this for you, they don't require to be explicitly aligned to a 16KB page size

Comment (zhaobozhen) [reply app/src/main/kotlin/com/absinthe/libchecker/features/chart/impl/PageSize16KBChartDataSource.kt:48]: Yes, I didn’t consider that. Maybe I should revise the copy and change “Not Supported” in the statistics categories to “No Native Library or Not Supported.” Would that be more accurate?

Comment (JunkFood02) [reply app/src/main/kotlin/com/absinthe/libchecker/features/chart/impl/PageSize16KBChartDataSource.kt:48]: maybe we can refer to the ABI chart and add a category for "no native code"?

Comment (zhaobozhen) [reply app/src/main/kotlin/com/absinthe/libchecker/features/chart/impl/PageSize16KBChartDataSource.kt:48]: Also a good idea. Maybe I will do it in next build.

**Pull request merged**